src/Command/ProcessCommand.php

```diff
@@ -57,8 +58,17 @@ final class ProcessCommand extends Command
 
     private int $maximumNumberOfTries;
 
+    /** @var ImageResource[] */
+    private array $processableImageResources = [];
```

Comment:
I think normally I wouldn't declare a class variable to hold this kind of value - I would probably give the value as an argument to the methods that needs it

Reply:
Well, that might be. I did this to allow for these things to be "processed" in the initialize method

Reply:
It's up to you :D